### PR TITLE
CLB-10 

### PR DESCRIPTION
**Pull request by bojanab987**
Add ProjectCard Component 
Include style resets in index.css
Format files by prettier

### CODE REVIEW

File: src/components/ProjectCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+}: Props) {
+    let statusClass = null;
+
+    if (status === 'active') {
+        statusClass = styles.active;
+    } else if (status === 'inactive') {
+        statusClass = styles.inactive;
+    } else if (status === 'cancelled') {
+        statusClass = styles.cancelled;
+    }
```
Comment (kimnovak) [inline src/components/ProjectCard/index.tsx]:
```suggestion
    statusClass = styles[status];
```

File: src/components/ProjectCard/types.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export type Props = {
+    status: 'active' | 'cancelled' | 'inactive';
```
Comment (kimnovak) [inline src/components/ProjectCard/types.ts]: Can you please try with enum
```suggestion
enum ProjectStatus {
     active = 'active',
     cancelled = 'cancelled',
     inactive = 'inactive',
 } 
export type Props = {
    status: ProjectStatus;
```

Comment (SreckoStojic) [reply src/components/ProjectCard/types.ts]: @kimnovak Good idea. I agree with you.

File: src/components/ProjectCard/index.tsx (hunk 1 above)
Comment (bojanab987) [reply src/components/ProjectCard/index.tsx]: Thanks for your comment! :) 
Changed as you suggested

File: src/components/ProjectCard/types.ts (hunk 2 above)
Comment (bojanab987) [reply src/components/ProjectCard/types.ts]: Thanks! 
I've changed this, too. 👍

**Pull request merged**